src/api/DataIntegrationApi.js

```diff
@@ -107,3 +108,39 @@ export function createEntityAndAssociationData(bulkDataCreation :Object) :Promis
       return Promise.reject(error);
     });
 }
+/**
+ * `POST /integration`
+ *
+ * Creates entities linked by associations for the given entity data.
+ *
+ * @static
+ * @memberof lattice.DataIntegrationApi
+ * @param {Object} entityKeys
+ * @return {Promise} - a Promise that resolves with a set of the entityKeyIds that were created for the given entityIds
+ *
+ * @example
+ *
+ * entityKey_1 = { entitySetId: "entitySetId_1", entityId: "entityId_1" };
+ * entityKey_2 = { entitySetId: "entitySetId_2", entityId: "entityId_2" };
+ *
+ * DataIntegrationApi.getEntityKeyIds({ entityIds: [entityKey_1, entityKey_2, ...] });
+ */
+export function getEntityKeyIds(entityKeys :string[]) :Promise<*> {
```

Comment:
`entityKeys` is not a string array. This should be `:Object[]`, or if you want to be fancy...

```
type EntityKey = {|
  entityId :string;
  entitySetId :UUID;
|};
export function getEntityKeyIds(entityKeys :EntityKey[]) :Promise<*> {
  ...
}
```